mkdocs.yml

```diff
@@ -1,4 +1,4 @@
-site_name: Structured Q&A
```

Comment:
In document to podcast, [you updated](https://github.com/mozilla-ai/document-to-podcast/commit/97617d45f46eeb5d679c12353043220c4468a6d3) it to use the actual repo name. Is there a reason to not follow the same pattern here?